server/test/conversations.spec.js

```diff
@@ -0,0 +1,76 @@
+require('dotenv').config();
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const app = require('../app.js');
+const { createUsers } = require('../utils/testUtils');
+
+const should = chai.should();
+chai.use(chaiHttp);
+
+let users = [];
+let singleConversationId = '';
+
+before('Create Users', (done) => {
+  users = createUsers(2);
+  done();
+});
+
+const checkSingleConversation = (err, res) => {
+  should.not.exist(err);
+  res.should.have.status(200);
+
+  res.body.should.contain.all.keys([
```

Comment:
The tests should check for correct values in addition to keys. I think I'm just seing that for users. Is that right?

Reply:
Well maybe types, but values should be based on the conversation created, IE changing every time